scripts/uberenv/spack_configs/toss_3_x86_64_ib/compilers.yaml

```diff
@@ -1,31 +1,33 @@
 compilers:
 - compiler:
+    # the "rh/usr" added to the path is a temporary work around until
```

Comment:
gyllenhall is actually propagating this fix for me now but I wasn't sure how long it would take him to do.